### DISCUSSION TITLE
It is a good practice to run this app all the time in production environment?

### DISCUSSION THREAD

**Discussion by blaisewang** [gojue/ecapture#135] [Q&A] [answered]
Hey, 

I am new to eBPF. I would like to run this app all the time with TLS or MySQL query sniff enabled for auditing. If the performance compromise is acceptable, what else should I take care of?

Thanks,
Blaise

**Comment by cfc4n** [comment, marked as answer]
Complete a performance test to make better decisions. 

And print event to console or logfile is not a best way.   You can fork it , collect or upload events  by yourself. For example, upload to ElasticSearch, prometheus, etc.
